Mage.Tests/src/test/java/org/mage/test/cards/copy/CopySpellTest.java

```diff
@@ -778,6 +777,80 @@ public void copySpellWithKicker() {
         assertPermanentCount(playerB, "Expedition Map", 0);
     }
 
+    /**
+     * Reported bug: https://github.com/magefree/mage/issues/11581
+     * Neverwinter Hydra is copied by Magus Lucea Kane's ability, but the copied version does not enter with +1/+1 counters.
+     */
+    @Test
+    public void test_CopyNeverwinterHydra() {
+        addCard(Zone.BATTLEFIELD, playerA, "Tropical Island", 4 + 2);
+        // <i>Psychic Stimulus</i> &mdash; {T}: Add {C}{C}. When you next cast a spell with {X} in its mana cost
+        // or activate an ability with {X} in its activation cost this turn, copy that spell or ability.
+        // You may choose new targets for the copy.
+        addCard(Zone.BATTLEFIELD, playerA, "Magus Lucea Kane");
+        addCard(Zone.HAND, playerA, "Neverwinter Hydra");
+        // Copy target creature spell you control, except it isn’t legendary if the spell is legendary.
+        addCard(Zone.HAND, playerA, "Double Major");
+
+        activateManaAbility(1, PhaseStep.PRECOMBAT_MAIN, playerA, "<i>Psychic");
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Neverwinter Hydra");
+        setChoice(playerA, "X=2");
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Double Major", "Neverwinter Hydra");
+
+        setStopAt(1, PhaseStep.BEGIN_COMBAT);
```

Comment:
please include `setStrictChooseMode(true)` for all new tests to be more robust. Probably have to add a few `addTarget` and `setChoice` to clarify what's happening.